public_suffix_list.dat

```diff
@@ -6526,7 +6526,7 @@ lib.ca.us
 lib.co.us
 lib.ct.us
 lib.dc.us
-lib.de.us
+// lib.de.us - Moved to Private section at request of Ed Moore <Ed.Moore@lib.de.us>
```

Comment:
For consistency with the other removals (e.g. 6463, 6454, 6423, etc) seems like this 
1) Should use two spaces after the entry (omit the -)
2) Include the Issue # that requested this ( https://github.com/publicsuffix/list/issues/243 - presumably "Issue #243" in code)
3) followed by -

that is
// lib.de.us  Issue #243 - Moved to PRIVATE at request of Ed Moore <Ed.Moore@lib.de.us>

wdyt?